mteb/tasks/PairClassification/multilingual/XStance.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks import MultilingualTask
+from ....abstasks.AbsTaskPairClassification import AbsTaskPairClassification
+
+
```

Comment:
`N_SAMPLES = 2048` 

I prefer adding here 

then  just pass it to

`     n_samples={"test": N_SAMPLES},
`

` def dataset_transform(self):
        max_n_samples = N_SAMPLES`